doc/source/serve/production-guide/index.md

```diff
@@ -0,0 +1,52 @@
+(serve-in-production)=
+
+# Production Guide
+
+The recommended way to run Ray Serve in production is on Kubernetes using the [KubeRay](kuberay-quickstart) [RayService] custom resource.
+The RayService custom resource automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
+If you're not running on Kubernetes, you can also run Ray Serve on a Ray cluster directly using the Serve CLI.
+
+This section should help you:
+
+- understand the [Serve config file format](serve-in-production-config-file).
+- understand how to deploy on VMs using the [Serve CLI](serve-in-production-deploying).
+- understand how to deploy on Kubernetes using a [`KubeRay` `RayService`](serve-in-production-kubernetes).
+- understand how to [monitor running Serve applications](serve-monitoring).
+
+(serve-in-production-example)=
+
+## Working example: FruitStand application
+
+Throughout the production guide, we will use the following Serve application as a working example.
+The application takes in requests containing a list of two values, a fruit name and an amount, and returns the total price for the batch of fruits.
+
+```{literalinclude} ../serve/doc_code/production_fruit_example.py
+:language: python
+:start-after: __fruit_example_begin__
+:end-before: __fruit_example_end__
+```
+
+This code is saved in the `fruit.py` file and the final deployment graph that we want to deploy is assigned to the `deployment_graph` variable.
+To run and test the application locally, use the `serve run fruit:deployment_graph` command:
+
+```console
+# Terminal window 1:
+
+$ ls
+fruit.py
+
+$ serve run fruit:deployment_graph
+2022-06-21 13:07:01,966  INFO scripts.py:253 -- Deploying from import path: "fruit:deployment_graph".
+2022-06-21 13:07:03,774  INFO services.py:1477 -- View the Ray dashboard at http://127.0.0.1:8265
+...
+2022-06-21 13:07:08,076  SUCC scripts.py:266 -- Deployed successfully.
+
+```
+
+We can test this graph by opening a new terminal window and making requests to it using `curl`.
+
+```console
+# Terminal window 2:
+
+$ curl -H "Content-Type: application/json" -d '{"PEAR": 2}' "http://localhost:8000/"
+```
```

Comment:
It's nice to end index pages with instructions on where to go next so the reader isn't left wondering. I would suggest repeating the links from the top of the page with a short prefix to explain the user persona:

1. To learn more about configuring your deployment, go to the [config file page].
2. ...

Reply:
good idea